fixtures/chauvet-dj/slimpar-pro-rgba.json

```diff
@@ -0,0 +1,276 @@
+{
+  "$schema": "https://raw.githubusercontent.com/OpenLightingProject/open-fixture-library/master/schemas/fixture.json",
+  "name": "SlimPAR Pro RGBA",
+  "categories": ["Color Changer"],
+  "meta": {
+    "authors": ["Ken Harris"],
+    "createDate": "2022-03-21",
+    "lastModifyDate": "2022-03-21"
+  },
+  "links": {
+    "manual": [
+      "https://www.chauvetdj.com/wp-content/uploads/2015/12/SlimPAR_Pro_RGBA_UM_Rev5_WO.pdf"
+    ],
+    "productPage": [
+      "https://www.chauvetdj.com/products/slimpar-pro-rgba/"
+    ],
+    "video": [
+      "https://www.youtube.com/watch?v=sXaC7ZqkGy4"
+    ]
+  },
+  "physical": {
+    "dimensions": [256, 290, 87],
+    "weight": 2.8,
+    "power": 90,
+    "DMXconnector": "3-pin",
+    "bulb": {
+      "type": "42× 1W RGBA LED"
+    },
+    "lens": {
+      "degreesMinMax": [21, 21]
+    }
+  },
+  "availableChannels": {
+    "Red": {
+      "capability": {
+        "type": "ColorIntensity",
+        "color": "Red",
+        "brightnessStart": "0%",
+        "brightnessEnd": "100%"
+      }
+    },
+    "Green": {
+      "capability": {
+        "type": "ColorIntensity",
+        "color": "Green",
+        "brightnessStart": "0%",
+        "brightnessEnd": "100%"
+      }
+    },
+    "Blue": {
+      "capability": {
+        "type": "ColorIntensity",
+        "color": "Blue",
+        "brightnessStart": "0%",
+        "brightnessEnd": "100%"
+      }
+    },
+    "Amber": {
+      "capability": {
+        "type": "ColorIntensity",
+        "color": "Amber",
+        "brightnessStart": "0%",
+        "brightnessEnd": "100%"
+      }
+    },
+    "Dimmer": {
+      "capability": {
+        "type": "Intensity",
+        "brightnessStart": "0%",
+        "brightnessEnd": "100%"
+      }
+    },
+    "Color Macros": {
+      "capabilities": [
+        {
+          "dmxRange": [0, 10],
+          "type": "NoFunction"
+        },
+        {
+          "dmxRange": [11, 200],
+          "type": "Generic",
+          "comment": "\"Color macros\""
```

Comment:
This probably should have been helpWanted -- there's no documentation of what these 190 "color macros" might be.

Reply:
I added the color presets. The `~` here refers to "changing value between start and end of this capability".